packages/twenty-server/src/modules/webhook/standard-objects/webhook.workspace-entity.ts

```diff
@@ -31,12 +31,12 @@ export class WebhookWorkspaceEntity extends BaseWorkspaceEntity {
 
   @WorkspaceField({
     standardId: WEBHOOK_STANDARD_FIELD_IDS.operation,
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.ARRAY,
     label: 'Operation',
     description: 'Webhook operation',
     icon: 'IconCheckbox',
   })
-  operation: string;
+  operations: string[];
```

Comment:
I think we should proceed with a 2 steps strategy to migrate data properly:
### First PR
- leave `operation` column like it exists now and create a new column `operations`
- create the command to copy data from `operation` to `operations` with arrays of lenght 1
- duplicate `operation` updates into the `operations` column
### Second PR
- update the frontend and backend so it uses `operations` instead of `operation`
- add the frontend code that enable `operations` arrays of length > 1
- remove the `operation` column